src/main/java/uk/gov/hmcts/reform/sscs/json/RoboticsJsonMapper.java

```diff
@@ -71,6 +82,13 @@ private static JSONObject buildAppealDetails(JSONObject obj, Appeal appeal, Stri
         return obj;
     }
 
+    private static String getCaseCode(String code) {
+        if (StringUtils.lowerCase(code).equals("esa")) {
```

Comment:
StringUtils.equalsIgnoreCase is slightly better

Reply:
nice one! I'll update it. Thanks